docs/plugins/react-native-mmkv.md

```diff
@@ -10,20 +10,25 @@ The `react-native-mmkv` plugin allows you to track [MMKV](https://github.com/mro
 
 Import your mmkv storage instance:
 
-```js
+```tsx
 import { MMKV } from "react-native-mmkv";
 export const storage = new MMKV();
 ```
 
 To use the `mmkvPlugin`, add the additional plugin on the `import` line.
 
-```js
+```tsx
 import Reactotron from "reactotron-react-native"
+// success-line
+import type { ReactotronReactNative } from "reactotron-react-native"
+// success-line
```

Comment:
What is this `// success-line` comment?

Reply:
It highlights the next line green in docusauras code blocks.

See this cookbook PR for more context: https://github.com/infinitered/ignite-cookbook/pull/139